### PR TITLE
[DOC] Kafka Exporter alerts

### PR DESCRIPTION
**Pull request by PaulRMellor**
Signed-off-by: prmellor <pmellor@redhat.com>

### Type of change
- Documentation

### Description
Update to Kafka Exporter content to include descriptions of the Prometheus alerts.
Also includes reference to the dashboard JSON file. 

### Checklist

_Please go through this checklist and make sure all applicable tasks have been done_

- [ ] Update/write design documentation in `./design`
- [ ] Write tests
- [ ] Make sure all tests pass
- [x] Update documentation
- [ ] Check RBAC rights for Kubernetes / OpenShift roles
- [ ] Try your changes from Pod inside your Kubernetes and OpenShift cluster, not just locally
- [ ] Reference relevant issue(s) and close them after merging
- [ ] Update CHANGELOG.md

### CODE REVIEW

**Comment by PaulRMellor** [comment]
@stanlyDoge, @scholzj  - please review

File: documentation/book/kafka-exporter/con-kafka-exporter-alerts.adoc [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+The rules for Kafka Exporter are defined in `prometheus-rules.yaml`, and are deployed with Prometheus.
+For more information, see  xref:assembly-metrics-prometheus-{context}[Prometheus].
+
+The alert notification rules specific to Kafka Exporter are as follows:
+
+`UnderReplicatedPartition`:: An alert to warn that a topic is under-replicated and the broker is not replicating enough partitions.
+A planned restart of the Kafka broker may be required to restart the replication process.
+
+`TooLargeConsumerGroupLag`:: An alert to warn that the lag on a consumer group is too big for a specific topic partition.
```
Comment (stanlyDoge) [inline documentation/book/kafka-exporter/con-kafka-exporter-alerts.adoc]: I would mention the default value here too. If I am not wrong, it should be '1000'.

File: documentation/book/kafka-exporter/con-kafka-exporter-alerts.adoc [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+[id='con-metrics-kafka-exporter-alerts{context}']
+
+= Kafka Exporter alerting rule examples
+
+If you performed the steps to introduce metrics to your deployment, you will already have your Kafka cluster configured to use the alert notification rules that support Kafka Exporter.
+
+The rules for Kafka Exporter are defined in `prometheus-rules.yaml`, and are deployed with Prometheus.
+For more information, see  xref:assembly-metrics-prometheus-{context}[Prometheus].
+
+The alert notification rules specific to Kafka Exporter are as follows:
```
Comment (scholzj) [inline documentation/book/kafka-exporter/con-kafka-exporter-alerts.adoc]: I think we should add that these rules need to be adapted to correspond to the user use-cases, scenarios, etc.

File: documentation/book/kafka-exporter/con-kafka-exporter-alerts.adoc [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+
+`UnderReplicatedPartition`:: An alert to warn that a topic is under-replicated and the broker is not replicating enough partitions.
+A planned restart of the Kafka broker may be required to restart the replication process.
+
+`TooLargeConsumerGroupLag`:: An alert to warn that the lag on a consumer group is too big for a specific topic partition.
+The lag might indicate that the number of partitions in the consumer group is insufficient or a crash or an unplanned shutdown has resulted in too much old data being consumed.
```
Comment (scholzj) [inline documentation/book/kafka-exporter/con-kafka-exporter-alerts.adoc]:
```suggestion
Too big consumer lag means that consumers are too slow and are falling behind the producers.
```

File: documentation/book/kafka-exporter/con-kafka-exporter-alerts.adoc (hunk 2 above)
Comment (PaulRMellor) [reply documentation/book/kafka-exporter/con-kafka-exporter-alerts.adoc]: Added: 'Adapt the default configuration of these rules according to your specific needs.'

File: documentation/book/kafka-exporter/con-kafka-exporter-alerts.adoc [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+
+The rules for Kafka Exporter are defined in `prometheus-rules.yaml`, and are deployed with Prometheus.
+For more information, see  xref:assembly-metrics-prometheus-{context}[Prometheus].
+
+The alert notification rules specific to Kafka Exporter are as follows:
+
+`UnderReplicatedPartition`:: An alert to warn that a topic is under-replicated and the broker is not replicating enough partitions.
+A planned restart of the Kafka broker may be required to restart the replication process.
+
+`TooLargeConsumerGroupLag`:: An alert to warn that the lag on a consumer group is too big for a specific topic partition.
```
Comment (laidan6000) [inline documentation/book/kafka-exporter/con-kafka-exporter-alerts.adoc]:
```suggestion
`TooLargeConsumerGroupLag`:: An alert to warn that the lag on a consumer group is too large for a specific topic partition.
```

File: documentation/book/kafka-exporter/con-kafka-exporter-alerts.adoc [hunk 5]
```diff
@@ -0,0 +1,31 @@
...
+= Kafka Exporter alerting rule examples
+
+If you performed the steps to introduce metrics to your deployment, you will already have your Kafka cluster configured to use the alert notification rules that support Kafka Exporter.
+
+The rules for Kafka Exporter are defined in `prometheus-rules.yaml`, and are deployed with Prometheus.
+For more information, see  xref:assembly-metrics-prometheus-{context}[Prometheus].
+
+The alert notification rules specific to Kafka Exporter are as follows:
+
+`UnderReplicatedPartition`:: An alert to warn that a topic is under-replicated and the broker is not replicating enough partitions.
```
[the remark this answers is not in the document]
Comment (laidan6000) [reply documentation/book/kafka-exporter/con-kafka-exporter-alerts.adoc]: Can you provide information on the default configuration?

[the remark this answers is not in the document]
Comment (PaulRMellor) [reply documentation/book/kafka-exporter/con-kafka-exporter-alerts.adoc]: How about: 
The alert might signify that a Kafka instance is down or the cluster is overloaded.

The alert will actually trigger for 1 or more missing partitions, so:
The default configuration is for one or missing partitions for a topic.

**Comment by PaulRMellor** [comment]
@tombentley @scholzj @ppatierno -- ready to merge

**Pull request merged**